isNice/solution.js

```diff
@@ -0,0 +1,7 @@
+function isNice(arr){
+  if (!arr.length) return false;
```

Comment:
It's generally considered bad practice not to use curly braces on your conditionals. This was the cause of a major Apple bug a few years back: https://www.wired.com/2014/02/gotofail/